homeassistant/components/scrape/config_flow.py

```diff
@@ -129,7 +129,15 @@
     ),
     vol.Required(CONF_UNIT_OF_MEASUREMENT, default=NONE_SENTINEL): SelectSelector(
         SelectSelectorConfig(
-            options=[NONE_SENTINEL] + sorted([cls.value for cls in UnitOfTemperature]),
+            options=[NONE_SENTINEL]
+            + sorted(
+                {
+                    str(unit)
+                    for units in SENSOR_DEVICE_CLASS_UNITS.values()
```

Comment:
This is only a list of units that have a device class mapping. These are not all available units.